p02_3.py

```diff
@@ -0,0 +1,30 @@
+def mean(arr):
+	sum = 0
+	l = len(arr)
+	if l == 0:
+		print("There are no numbers in the array!")
```

Comment:
What is the purpose of this output?
if you want to give error message, you should `raise`
PS. should make the error message clearer, like `"Array is empty"`